global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
global_controller - log_exception(line:828) - Exception on /proxyLoad [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1164, in make_response
    raise TypeError(
TypeError: The view function for 'handleProxyLoad' did not return a valid response. The function either returned None or ended without a return statement.
__main__ - filter_incomplete_traces(line:1292) - !!! ERROR: len(complete_traces[us-east-1]) == 0
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(588) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 598
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(790) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 800
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(990) > total capcity(800)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[b][us-east-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,b, 400 -> 505
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(818) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 828
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(990) > total capcity(905)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[b][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,b, 400 -> 505
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(818) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 828
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(1208) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 1218
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(557) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 567
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(618) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 628
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(596) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 606
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(634) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 644
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(545) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 555
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(562) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 572
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(590) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 600
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(634) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 644
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(606) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 616
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(590) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 600
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(597) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 607
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(570) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 580
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(564) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 574
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(590) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 600
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(881) > total capcity(800)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[b][us-east-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,b, 400 -> 450
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(601) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 611
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(881) > total capcity(850)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[b][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,b, 400 -> 450
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(641) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 651
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(581) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 591
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(567) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 577
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(565) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 575
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(550) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 560
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(491) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 501
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(564) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 574
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(598) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 608
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(695) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 705
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(630) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 640
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(601) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 611
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(622) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 632
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(570) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 580
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(527) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 537
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(543) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 553
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(530) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 540
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(517) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 527
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(481) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 491
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(435) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 445
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(504) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 514
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(574) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 584
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(535) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 545
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(528) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 538
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(530) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 540
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(532) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 542
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(519) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 529
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(527) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 537
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(503) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 513
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(502) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 512
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(537) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 547
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(480) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 490
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(498) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 508
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(521) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 531
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(534) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 544
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(527) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 537
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(527) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 537
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(531) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 541
__main__ - optimizer_entrypoint(line:877) - !!! ERROR !!! Total demand(504) > total capcity(400)
__main__ - optimizer_entrypoint(line:878) - max_capacity_per_service[c][us-west-1]: 400
__main__ - optimizer_entrypoint(line:880) - new_capacity,c, 400 -> 514
